lib/plugins/aws/customResources/resources/utils.js

```diff
@@ -56,13 +56,22 @@ function response(event, context, status, data = {}, err) {
 }
 
 function getLambdaArn(region, accountId, functionName) {
-  return `arn:aws:lambda:${region}:${accountId}:function:${functionName}`;
+  let awsNode = 'aws';
+  if (region === 'us-gov-west-1' || region === 'us-gov-east-1') {
+    awsNode = 'aws-us-gov';
+  }
+  return `arn:${awsNode}:lambda:${region}:${accountId}:function:${functionName}`;
```

Comment:
I believe we should rather rely on [AWS::Partition](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html#cfn-pseudo-param-partition) as e.g. we already do here: https://github.com/serverless/serverless/blob/c77cf010718a243bb98b922c4b9ba7e6c6318a62/lib/plugins/aws/customResources/index.js#L140

As there are other region cases which require that (e.g. china regions)

Reply:
Ok I can make this change to utilize the AWS partition

Reply:
Actually, in reading the the code that uses these utils functions, I do not believe we have access to leverage the `AWS::Partition`?

When this code invoked, we are in a Lambda execution context, not in CloudFormation templating/execution context.

Reply:
Indeed, in that case we need to somehow reliably resolve a partition for current region.
Still at this point I'm not sure how to do that (I see it's not available as env variable)

Reply:
You should be able to gather the AWS partition from the `invokedFunctionArn` in the [context](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html) object, similar to how we can get the partition [here](https://github.com/serverless/serverless/blob/master/lib/plugins/aws/provider/awsProvider.js#L516)

Reply:
Nice, ok I'll try that.  I probably won't be able to get it done today, but this weekend hopefully.  Thanks!